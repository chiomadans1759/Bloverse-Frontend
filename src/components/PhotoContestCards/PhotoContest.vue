<template>
  <main class="container-fluid bl-photoContest">
    <div class="bl-photoContest--header">
      <p class="font-weight-bold title">Photo Contest</p>
      <p class="subtitle">
        Lorem ipsum dolor sit amet, vel accumsan
      </p>
    </div>
    <carousel
      :perPage = "perPage"
      navigationEnabled
      paginationActiveColor = "#096DD9"
      paginationColor = "#868686"
      :paginationSize = "paginationSize"
      :paginationPadding = "paginationPadding"
      :minSwipeDistance = "minSwipeDistance"
      :navigationNextLabel = "navigationNextLabel"
      :navigationPrevLabel = "navigationPrevLabel"
      :autoplayTimeout="4000"
      :autoplay = "autoplay"
      :loop = "loop"
    >
      <slide>
        <photo-contest-card />
      </slide>
      <slide>
        <photo-contest-card />
      </slide>
      <slide>
        <photo-contest-card />
      </slide>
      <slide>
        <photo-contest-card />
      </slide>
      <slide>
        <photo-contest-card />
      </slide>
      <slide>
        <photo-contest-card />
      </slide>
      <slide>
        <photo-contest-card />
      </slide>
    </carousel>
  </main>
</template>

<script>
import { Carousel, Slide } from 'vue-carousel';
import PhotoContestCard from './PhotoContestCard';

export default {
  name: 'PhotoContest',
  components: { Carousel, Slide, PhotoContestCard },
  data() {
    return {
      perPage: 3,
      paginationSize: 5,
      paginationPadding: 5,
      minSwipeDistance: 0,
      autoplay: true,
      loop: true,
      navigationNextLabel: `
      <span style="font-size: 20px; color: #000000; cursor: pointer;">
        <i class="fal fa-caret-circle-right"></i>
      </span>
      `,
      navigationPrevLabel: `
      <span style="font-size: 20px; color: #000000; cursor: pointer;">
        <i class="fal fa-caret-circle-left"></i>
      </span>
      `,
    }
  }
}
</script>

<style lang='scss' scoped>
@import './Photocontest.scss';
</style>
