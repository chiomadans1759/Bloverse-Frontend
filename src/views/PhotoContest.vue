<template>
  <main id="photo-contest">
    <photo-contest-header/>

    <section
      class="text-center text-white my-0 position-relative d-flex justify-content-center align-items-center top-section">
      <div class>
        <div class="overlay"></div>
        <img class="img-fluid w-100" src="@/assets/photo5.jpg" alt>
      </div>
      <div class="container" style="z-index: 1;">
        <div class="row">
          <div class="col-md-12">
            <div class="mb-5">
              <p class="display-2 font-weight-normal mb-3">Bloverse Photo Contest</p>
              <p class="lead">‘The Story Behind The Photo’</p>
              <router-link to="/creators/apply" class="btn btn-primary mt-5">Enter Photo Contest</router-link>
            </div>
            <div class="pt-3">
              <button @click.prevent="showVid()" class="btn rounded-circle btn-play">
                <i class="fas fa-play"></i>
              </button>
              <span class="ml-3">Watch Video</span>
            </div>

            <video-modal
              v-if="show_video"
              @close="closeVid"
              source="https://www.youtube.com/embed/CjKVWRErsN8" />
          </div>
        </div>
      </div>
    </section>

    <section class="detail-wrap">
      <div class="container">
        <div class="row justify-content-center align-items-center detail-content">
          <div class="col-sm-10 col-md-9 col-lg-5 pr-lg-4">
            <div class="detail">
              <h1 class="title">How to enter:</h1>
              <ol>
                <li>
                  <router-link to="/creators">Apply</router-link> as a content creator
                </li>
                <li>Submit ‘Story Behind The Photo’ post</li>
                <li>Share on IG, FB, Twitter (take a screenshot of your article and upload to your IG account, put link in post comments or Bio)</li>
                <li>Include hashtags #JointheBloverse #Bloverse #bloverse{country} e.g #bloverseNigeria</li>
                <li>Tag our official handle while you are sharing @bloverse on IG and FB, and @bloverse_world on Twitter</li>
                <li>Tag three other content creators while sharing</li>
                <li>Follow us on social media (@bloverse on IG, @bloverse_world on twitter, and @bloverse on FB)</li>
              </ol>
            </div>
          </div>
          <div class="col-lg-6 d-none d-xl-block d-lg-block">
            <div class="d-flex justify-content-center image-wrap">
              <img class="img-fluid w-100" src="@/assets/photo6.jpg" alt>
            </div>
          </div>
        </div>
        <div class="row justify-content-center align-items-center detail-content">
          <div class="col-lg-6 d-none d-xl-block d-lg-block">
            <div class="d-flex justify-content-center image-wrap">
              <img class="img-fluid w-100" src="@/assets/photo3.png" alt>
            </div>
          </div>
          <div class="col-sm-10 col-md-9 col-lg-5 pl-lg-4">
            <div class="detail">
              <h1 class="title">Judging Criteria:</h1>
              <p>Each view is worth 1 point.</p>
              <p>All publications are ranked based on the total number of points accrued over the course of the contest.</p>
            </div>
          </div>
        </div>
        <div class="row justify-content-center align-items-center detail-content reward">
          <div class="col-lg-3 col-md-5 col-sm-6">
            <div class="card text-center gold mb-0">
              <div class="card-body p-0 pb-5">
                <img class src="@/assets/gold-medal.svg" alt>
                <div class="mt-4">
                  <div class="card-title font-weight-bold">1st Price</div>
                  <div class="display-1 font-weight-normal">$150</div>
                </div>
              </div>
            </div>
          </div>
          <div class="col-lg-3 col-md-5 col-sm-6">
            <div class="card text-center silver mb-0">
              <div class="card-body p-0 pb-5">
                <img class src="@/assets/Silver-medal.svg" alt>
                <div class="mt-4">
                  <div class="card-title font-weight-bold">2nd Price</div>
                  <div class="display-1 font-weight-normal">$100</div>
                </div>
              </div>
            </div>
          </div>
          <div class="col-lg-3 col-md-5 col-sm-6">
            <div class="card text-center bronze mb-0">
              <div class="card-body p-0 pb-5">
                <img class src="@/assets/bronze-medal.svg" alt>
                <div class="mt-4">
                  <div class="card-title font-weight-bold">3rd Price</div>
                  <div class="display-1 font-weight-normal">$50</div>
                </div>
              </div>
            </div>
          </div>
          <div class="col-lg-3 col-md-5 col-sm-6">
            <div class="card text-center platinum mb-0">
              <div class="card-body p-0 pb-5">
                <img class src="@/assets/platinum-medal.svg" alt>
                <div class="mt-4">
                  <div class="card-title font-weight-bold">Highest Ranked Creator</div>
                  <div class="display-1 font-weight-normal">$100</div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    
    <section>
      <div class="container">
        <div class="row justify-content-center">
          <div class="col-lg-8 col-md-10">
            <p
              class="mb-5"
            >Contest is open to all residents of Australia, Mexico, United States, Canada, United Kingdom, Ireland, Italy, France, Germany, Netherlands, United Arab Emirates, Pakistan, India, Singapore, Philippines, Hong Kong, Kenya, South Africa, Nigeria and Ghana.</p>
            <div class="d-sm-flex align-items-center">
              <h3 class="font-weight-bold">Spread The Word:</h3>
              <ul class="list-unstyled social-round">
                <li>
                  <a
                    class="d-flex justify-content-center align-items-center rounded-circle facebook-color"
                    href="https://www.facebook.com/bloverse/"
                    target="_blank"
                  >
                    <i class="fab fa-facebook-f"></i>
                  </a>
                </li>
                <li>
                  <a
                    class="d-flex justify-content-center align-items-center rounded-circle twitter-color"
                    href="https://twitter.com/bloverse_world"
                    target="_blank"
                  >
                    <i class="fab fa-twitter"></i>
                  </a>
                </li>
                <li>
                  <a
                    class="d-flex justify-content-center align-items-center rounded-circle linkedin-color"
                    href="https://www.linkedin.com/company/bloverse1/"
                    target="_blank"
                  >
                    <i class="fab fa-linkedin-in"></i>
                  </a>
                </li>
                <li>
                  <a
                    class="d-flex justify-content-center align-items-center rounded-circle instagram-color"
                    href="https://www.instagram.com/bloverse"
                    target="_blank"
                  >
                    <i class="fab fa-instagram"></i>
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
        <the-footer/>
      </div>
    </section>
  </main>
</template>

<script>
import PhotoContestHeader from "@/components/PhotoContestHeader";
import TheFooter from "@/components/TheFooter";
import VideoModal from "@/components/VideoModal.vue";

export default {
  name: "photo-contest",
  components: {
    PhotoContestHeader,
    TheFooter,
    VideoModal
  },
  data() {
    return {
      show_video: false
    };
  },
  methods: {
    showVid() {
      this.show_video = true;
    },
    closeVid() {
      this.show_video = false;
    }
  }
};
</script>

<style scoped>
#photo-contest {
  font-size: 1rem;
  line-height: 1.6;
}
section {
  margin: 7rem 0;
}
.top-section {
  height: 38.75rem;
  overflow: hidden;
}
.top-section img {
  position: absolute;
  right: 0;
  left: 0;
  top: 0;
  bottom: 0;
  margin: auto;
  opacity: 0.35;
}
.btn-play {
  width: 4.5rem;
  height: 4.5rem;
}
.image-wrap {
  height: 23rem;
  border-radius: 1rem;
  overflow: hidden;
  border: rgba(0, 0, 0, 0.15);
  position: relative;
  background: #fff;
}
.image-wrap img {
  position: absolute;
  right: 0;
  left: 0;
  top: 0;
  bottom: 0;
  margin: auto;
}
.detail-wrap .detail-content {
  margin-bottom: 7rem;
}
.detail-wrap .detail-content:last-child {
  margin-bottom: 0;
}
.detail ol li,
.detail p,
.reward .card .card-title {
  margin-bottom: 0.5rem;
}
.detail ol li {
  padding-left: 0.5rem;
}
.detail ol {
  margin-left: 1rem;
}
.detail .title {
  margin-bottom: 1.5rem;
}
.detail h1.title {
  font-size: 1.875rem;
  color: #252525;
}
.overlay {
  height: 100%;
  width: 100%;
  position: absolute;
  left: 0;
  top: 0;
  background-color: #000;
  overflow-x: hidden;
}
.reward .card {
  border: 1px solid !important;
  border-radius: 0 !important;
  overflow: hidden;
  cursor: pointer;
  transition: all 0.25s ease-in-out;
}
.reward .card:hover {
  transform: scale(1.22);
  -webkit-transform: scale(1.22);
  z-index: 2;
}
.reward .card img {
  margin-top: -2px;
}
ul.social-round li {
  display: inline-block;
  margin-left: 1rem;
}
ul.social-round li:first-child {
  margin-left: 2rem;
}
ul.social-round li a {
  height: 3.5rem;
  width: 3.5rem;
  border: 1px solid rgba(0, 0, 0, 0.15);
  color: #ffffff;
}
.facebook-color {
  background-color: #3b5998;
}
.twitter-color {
  background-color: #38a1f3;
}
.linkedin-color {
  background-color: #0077b5;
}
.instagram-color {
  background-color: #fb3958;
}
.reward .card.silver {
  border-color: #bcbec0 !important;
}
.reward .card.gold {
  border-color: #e5b64e !important;
}
.reward .card.bronze {
  border-color: #d37b40 !important;
}
.reward .card.platinum {
  border-color: #096dd9 !important;
}
.card .display-1 {
  font-size: 3.5rem;
  color: #096dd9;
}
.cursor-pointer {
  cursor: pointer;
}

@media (max-width: 576px) {
  ul.social-round {
    margin-top: 1rem;
  }
  ul.social-round li:first-child {
    margin-left: 0;
  }
}
@media (max-width: 991.98px) {
  .reward .card {
    margin-bottom: 2rem !important;
  }
  section {
    margin: 3rem 0;
  }
  .detail-wrap .detail-content {
    margin-bottom: 3rem;
  }
  #photo-contest {
    padding-bottom: 2rem;
  }
}
</style>
