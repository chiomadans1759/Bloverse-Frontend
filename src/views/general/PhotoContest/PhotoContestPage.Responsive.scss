@media (max-width: 576px) {
  #bl-photocontest {
    overflow-x: hidden;
    .ongoing-contest-section {
      padding: 0 !important;

      .title-container {
        padding: rem(40px) rem(17px) rem(10px) rem(17px);
      }
    }

    .contestant-list {
      padding: rem(23px) rem(16px) rem(23px) rem(16px);
      width: auto;
      .badge-container {
        text-align: right;
      }
    }

    .judging-criteria {
      padding-top: rem(40px);
      padding-bottom: rem(40px);
      .criteria {
        width: auto;
      }
      .criteria-content {
        padding-left: rem(80px);
      }
    }

    .prize-section {
      padding-top: rem(40px);
      padding-bottom: rem(40px);

      .prize {
        margin-bottom: rem(17px);

        .card {
          .card-icon {
            width: auto;
          }
        }
      }
    }
  }
}

@media (max-width: 768px) {
  #bl-photocontest {
    .judging-criteria {
      padding-top: rem(40px);
      padding-bottom: rem(40px);
      .criteria {
        width: auto;
      }
      .criteria-content {
        padding-left: rem(80px);
      }
    }
  }
}
