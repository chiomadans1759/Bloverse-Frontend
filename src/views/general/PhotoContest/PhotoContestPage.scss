@import 'sass-rem';

#bl-photocontest {

  background-color: #ffffff;
  .top-section-container {
    background-color: #096DD9;
    margin: 0;
    background-image:
          linear-gradient(rgba(18, 18, 19, 0.8), rgba(18, 18, 19, 0.8)),
          url('../../../assets/photo-4.jpg');
    color: white;
    background-repeat: no-repeat;
    background-size: cover;
    background-position: center;

    .row {
      margin: 0;
    }

    .video-button {
      height: rem(80px);
      width: rem(80px);
      position: absolute;
      top: 50%;
      left: 50%;
      transform: translate(-50%, -50%);
      background-color: #ffffff;
      border-radius: 50%;

      .video-controls {
        position: absolute;
        top: 50%;
        left: 50%;
        transform: translate(-50%, -50%);
        color: #252525;
      }

      button {
        background-color: transparent;
        border: none;
      }
    }

    .title {
      font-size: rem(56px);
      padding: rem(10px);
    }
    
    .paragraph {
      font-size: rem(20px);
      padding: rem(10px);
    }
    
    .publish-link {
      background-image: linear-gradient( to right, #003CE1, #2F80ED);
      color: #ffffff;
      padding: rem(10px);
      font-size: rem(18px);
    }
    
    img {
      height: rem(400px);
      width: rem(600px);
      padding-top: rem(20px);
      padding-bottom: rem(20px);
    }
  }
  
  .publish-link-container {
    padding: rem(10px);
  }
  
  .ongoing-contest-section {
    padding: rem(40px) rem(50px) rem(40px) rem(50px);

    .title {
      font-size: rem(24px);
    }

    .paragraph {
      font-size: rem(14px);
      padding-top: rem(5px);
      color: #868686;
    }
  }

  .contestant-list {
    width: rem(100%);

    .contestant-title {
      font-size: rem(14px);
      font-weight: 600;

    }

    .contestant-filter {
      font-size: rem(14px);
    }

    .badge {
      background-color: transparent;
      width: rem(22px);
      height: rem(16px);
      color: #096DD9;
      border: rem(1px) solid #096DD9;
      border-radius: rem(2px);
    }

    .contestant-body {
      border: 1px solid rgba(0, 0, 0, 0.15);
      .contestant-row {
        padding-bottom: rem(10px);
      }
      .contestant-names {
        padding: rem(10px);
        height: rem(200px);
        overflow-y: scroll;
        overflow-x: hidden;
        .contestant-img {
          height: rem(40px);
          width: rem(40px);
          border-radius: rem(2px);
        }
        .contestant-name {
          padding: rem(10px) rem(10px) rem(10px) rem(20px);
          font-weight: 900;
        }
      }
    }

    .contestant-bottom {
      border-top: rem(1px) solid rgba(0, 0, 0, 0.15);
      .contestant-link-container {
        padding: rem(10px);
        text-align: center;
        .contestant-link {
          background-image: linear-gradient( to right, #003CE1, #2F80ED);
          color: #ffffff;
          padding: rem(10px);
          font-size: rem(14px);
          width: 100%;
        }
      }
    }

  }

  .how-to-enter {
    border-top: 1px solid rgba(0, 0, 0, 0.15);
    padding-top: rem(64px);
    padding-bottom: rem(64px);

    .row {
      margin: 0;
    }

    .enter-image {
      width: rem(479px);
      height: rem(362.76px);
    }

    .left-side {
      padding-top: rem(24px);
    }

    .how-to-enter-text {
      font-size: rem(24px);
      font-weight: 900;
    }

    li {
      padding-left: rem(27px);
      padding-top: rem(16px);
      font-size: rem(16px);
    }

    .bold-text {
      font-weight: 900;
    }

    .steps {
      padding-left: rem(20px);
    }
  }

  .judging-criteria {

    .row {
      margin: 0;
    }

    padding-top: rem(64px);
    padding-bottom: rem(64px);
    .judging-image {
      width: rem(479px);
      height: rem(680px);
    }

    .judging-criteria-text {
      font-size: rem(24px);
      font-weight: 900;

    }

    .criteria-icon {
      position: absolute;
      background-color: #ffffff;
      left: rem(-20px);
      border: 1px solid rgba(0, 0, 0, 0.15);
      height: rem(72px);
      width: rem(72px);
      margin: auto;
      text-align: center;
      vertical-align: middle;
      .icon {
        color: #096DD9;
      }
      svg {
        position: absolute;
        left: 50%;
        top: 50%;
        transform: translate(-50%, -50%);
      }
    }

    .criteria-content {
      color: #565656;

      .criteria-text {
        font-size: rem(16px);
        font-weight: 900;
        padding-bottom: rem(8px);
      }

      p {
        font-size: rem(16px);
      }
    }

    .criteria {
      position: relative;
      padding-top: rem(24px);
      padding-bottom: rem(24px);
      border: rem(1px) solid rgba(0, 0, 0, 0.15);
      margin-bottom: rem(8px) ;
    }
  }

  .prize-section {
    padding-top: rem(64px);
    padding-bottom: rem(64px);
    text-align: center;

    .row {
      margin: 0;
    }

    .prize .card:hover {
      transform: scale(1.22);
      -webkit-transform: scale(1.22);
      z-index: 2;
    }

    .prize {
      cursor: pointer;
      .card {
        background-color: #F5F5F5;
        border-radius: 0 !important;
        border-bottom-width: rem(4px) !important;
        min-height: rem(290px);
        transition: all .25s ease-in-out;
        .card-title {
          font-size: rem(16px);
          color: #252525;
          font-weight: 900;
        }
        .card-text {
          font-size: rem(56px);
          color: #096DD9;
        }
        .card-icon {
          height: rem(100px);
          width: rem(100px);
        }

        .card-img-overlay {
          padding: 0;
          div {
            padding-top: rem(20px);
          }
        }
      }
      .gold-card {
        border-color: #E5B64E !important;
        background-image: url('../../../assets/gold-medal-mask.svg');
        background-repeat: no-repeat;
        background-size: cover;
        background-position: center;
      }
      .silver-card {
        border-color: #BCBEC0 !important;
        background-image: url('../../../assets/silver-medal-mask.svg');
        background-repeat: no-repeat;
        background-size: cover;
        background-position: center;
      }
      .bronze-card {
        border-color: #D37B40 !important;
        background-image: url('../../../assets/bronze-medal-mask.svg');
        background-repeat: no-repeat;
        background-size: cover;
        background-position: center;
      }
      .platinum-card {
        border-color: #ABD1E4 !important;
        background-image: url('../../../assets/platinum-medal-mask.svg');
        background-repeat: no-repeat;
        background-size: cover;
        background-position: center;
      }
    }
  }

  .footer-section {
    padding-top: rem(64px);
    background-color: #F5F5F5;

    .row {
      margin: 0;
    }

    .footer-text {
      font-size: rem(16px);
      color: #565656;
    }
    .word-spread {
      margin-top: rem(20px);
      .title {
        font-size: rem(14px);
        font-weight: 900;
      }

      .social-icon {
        position: relative;
        padding-left: rem(16px);

        img {
          height: rem(38px);
          width: rem(38px);
        }
      }
    }

    .copyright {
      margin-top: rem(64px);
      .title {
        font-size: rem(14px);
        color: #B7B7B7;

        span {
          padding-left: rem(4px);
          font-size: rem(10px);
        }
      }

      .sub-text {
        font-size: rem(10px);
        color: rgba(0, 0, 0, 0.45);
      }
    }
  }
}
