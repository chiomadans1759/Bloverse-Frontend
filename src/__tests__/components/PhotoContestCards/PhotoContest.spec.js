import { mount, RouterLinkStub  } from '@vue/test-utils'
import PhotoContest from '@/components/PhotoContestCards/PhotoContest.vue';

describe('PhotoContest Component', () => {
  it('should render correctly', () => {

    const wrapper = mount(PhotoContest, {
      stubs: {
        RouterLink: RouterLinkStub
      }
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
