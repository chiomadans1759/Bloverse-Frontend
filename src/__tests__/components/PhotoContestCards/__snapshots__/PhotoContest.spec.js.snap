// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoContest Component should render correctly 1`] = `
<main
  class="container-fluid bl-photoContest"
>
  <div
    class="bl-photoContest--header"
  >
    <p
      class="font-weight-bold title"
    >
      Photo Contest
    </p>
     
    <p
      class="subtitle"
    >
      
      Lorem ipsum dolor sit amet, vel accumsan
    
    </p>
  </div>
   
  <section
    class="VueCarousel"
  >
    <div
      class="VueCarousel-wrapper"
    >
      <div
        class="VueCarousel-inner"
        role="listbox"
        style="transform: translate(0px, 0); visibility: hidden; height: auto;"
      >
        <div
          class="VueCarousel-slide"
          tabindex="-1"
        >
          <main
            class="card text-white photoContestCard"
          >
            <img
              alt="trend image"
              class="card-img"
              src="@/assets/photo-contest-1.jpg"
            />
             
            <div
              class="card-img-overlay photoContestCard--overlay"
            >
              <div
                class="content photoContestCard--overlay__content"
              >
                <p
                  class="card-text font-weight-bold photoContestCard--text"
                >
                  
        Lorem ipsum dolor sit amet, vel accumsan
      
                </p>
                 
                <p
                  class="card-text photoContestCard--username"
                >
                  <a
                    class="btn btn-primary initials"
                    href="#"
                    role="button"
                  />
                   
                  <span
                    class="name"
                  >
                    John Doe
                  </span>
                </p>
              </div>
            </div>
          </main>
        </div>
         
        <div
          class="VueCarousel-slide"
          tabindex="-1"
        >
          <main
            class="card text-white photoContestCard"
          >
            <img
              alt="trend image"
              class="card-img"
              src="@/assets/photo-contest-1.jpg"
            />
             
            <div
              class="card-img-overlay photoContestCard--overlay"
            >
              <div
                class="content photoContestCard--overlay__content"
              >
                <p
                  class="card-text font-weight-bold photoContestCard--text"
                >
                  
        Lorem ipsum dolor sit amet, vel accumsan
      
                </p>
                 
                <p
                  class="card-text photoContestCard--username"
                >
                  <a
                    class="btn btn-primary initials"
                    href="#"
                    role="button"
                  />
                   
                  <span
                    class="name"
                  >
                    John Doe
                  </span>
                </p>
              </div>
            </div>
          </main>
        </div>
         
        <div
          class="VueCarousel-slide"
          tabindex="-1"
        >
          <main
            class="card text-white photoContestCard"
          >
            <img
              alt="trend image"
              class="card-img"
              src="@/assets/photo-contest-1.jpg"
            />
             
            <div
              class="card-img-overlay photoContestCard--overlay"
            >
              <div
                class="content photoContestCard--overlay__content"
              >
                <p
                  class="card-text font-weight-bold photoContestCard--text"
                >
                  
        Lorem ipsum dolor sit amet, vel accumsan
      
                </p>
                 
                <p
                  class="card-text photoContestCard--username"
                >
                  <a
                    class="btn btn-primary initials"
                    href="#"
                    role="button"
                  />
                   
                  <span
                    class="name"
                  >
                    John Doe
                  </span>
                </p>
              </div>
            </div>
          </main>
        </div>
         
        <div
          class="VueCarousel-slide"
          tabindex="-1"
        >
          <main
            class="card text-white photoContestCard"
          >
            <img
              alt="trend image"
              class="card-img"
              src="@/assets/photo-contest-1.jpg"
            />
             
            <div
              class="card-img-overlay photoContestCard--overlay"
            >
              <div
                class="content photoContestCard--overlay__content"
              >
                <p
                  class="card-text font-weight-bold photoContestCard--text"
                >
                  
        Lorem ipsum dolor sit amet, vel accumsan
      
                </p>
                 
                <p
                  class="card-text photoContestCard--username"
                >
                  <a
                    class="btn btn-primary initials"
                    href="#"
                    role="button"
                  />
                   
                  <span
                    class="name"
                  >
                    John Doe
                  </span>
                </p>
              </div>
            </div>
          </main>
        </div>
         
        <div
          class="VueCarousel-slide"
          tabindex="-1"
        >
          <main
            class="card text-white photoContestCard"
          >
            <img
              alt="trend image"
              class="card-img"
              src="@/assets/photo-contest-1.jpg"
            />
             
            <div
              class="card-img-overlay photoContestCard--overlay"
            >
              <div
                class="content photoContestCard--overlay__content"
              >
                <p
                  class="card-text font-weight-bold photoContestCard--text"
                >
                  
        Lorem ipsum dolor sit amet, vel accumsan
      
                </p>
                 
                <p
                  class="card-text photoContestCard--username"
                >
                  <a
                    class="btn btn-primary initials"
                    href="#"
                    role="button"
                  />
                   
                  <span
                    class="name"
                  >
                    John Doe
                  </span>
                </p>
              </div>
            </div>
          </main>
        </div>
         
        <div
          class="VueCarousel-slide"
          tabindex="-1"
        >
          <main
            class="card text-white photoContestCard"
          >
            <img
              alt="trend image"
              class="card-img"
              src="@/assets/photo-contest-1.jpg"
            />
             
            <div
              class="card-img-overlay photoContestCard--overlay"
            >
              <div
                class="content photoContestCard--overlay__content"
              >
                <p
                  class="card-text font-weight-bold photoContestCard--text"
                >
                  
        Lorem ipsum dolor sit amet, vel accumsan
      
                </p>
                 
                <p
                  class="card-text photoContestCard--username"
                >
                  <a
                    class="btn btn-primary initials"
                    href="#"
                    role="button"
                  />
                   
                  <span
                    class="name"
                  >
                    John Doe
                  </span>
                </p>
              </div>
            </div>
          </main>
        </div>
         
        <div
          class="VueCarousel-slide"
          tabindex="-1"
        >
          <main
            class="card text-white photoContestCard"
          >
            <img
              alt="trend image"
              class="card-img"
              src="@/assets/photo-contest-1.jpg"
            />
             
            <div
              class="card-img-overlay photoContestCard--overlay"
            >
              <div
                class="content photoContestCard--overlay__content"
              >
                <p
                  class="card-text font-weight-bold photoContestCard--text"
                >
                  
        Lorem ipsum dolor sit amet, vel accumsan
      
                </p>
                 
                <p
                  class="card-text photoContestCard--username"
                >
                  <a
                    class="btn btn-primary initials"
                    href="#"
                    role="button"
                  />
                   
                  <span
                    class="name"
                  >
                    John Doe
                  </span>
                </p>
              </div>
            </div>
          </main>
        </div>
      </div>
    </div>
     
    <div
      class="VueCarousel-pagination"
      data-v-438fd353=""
      style="display: none;"
    >
      <ul
        class="VueCarousel-dot-container"
        data-v-438fd353=""
        role="tablist"
      />
    </div>
     
    <!---->
  </section>
</main>
`;
